Process.h

```diff
@@ -242,7 +242,7 @@ typedef struct Process_ {
    /*
     * Internal state for tree-mode.
     */
-   int indent;
+   int64_t indent;
```

Comment:
Isn't this somewhat overkill? Given that this mostly gives a level on screen? `int32_t` should suffice.